### PR TITLE
feat: add support for displaying private key discreetly for `snarkos account new`

### PR DESCRIPTION
**Pull request by Halimao**
## Motivation

Currently, running `snarkos account new` will show private key dicrectly in current terminal, it would be better to output it in an alternate screen. See following pics

image
image

### CODE REVIEW

File: account/src/lib.rs [hunk 1]
```diff
@@ -146,11 +148,14 @@ impl<N: Network> FromStr for Account<N> {
 impl<N: Network> Display for Account<N> {
     /// Renders the account as a string.
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        display_string_discreetly(
```
Comment (howardwu) [inline account/src/lib.rs]: Let's add a CLI flag, such as `--discreet` to make this approach optional.

Comment (Halimao) [reply account/src/lib.rs]: Okay, I will add it later😊

Comment (Halimao) [reply account/src/lib.rs]: @howardwu Addressed

**Comment by Halimao** [comment]
@howardwu Hi sir, suggestions has addressed, looking forward for your review again

**Comment by Halimao** [comment]
@howardwu Hi sir, is there anything else need to add? Or can we get this merged then😁

File: cli/src/commands/account.rs [hunk 2]
```diff
@@ -39,32 +41,35 @@ pub enum Account {
         /// Try until an address with the vanity string is found
         #[clap(short = 'v', long)]
         vanity: Option<String>,
+        /// Print sensitive information(such as private key) discreetly to an alternate screen
```
Comment (howardwu) [inline cli/src/commands/account.rs]:
```suggestion
        /// Print sensitive information (such as the private key) discreetly in an alternate screen
```

**Review by howardwu — APPROVED** [review]

**Pull request merged**